README.md

```diff
@@ -1,198 +1,9 @@
-# ionic-mobile-open-messaging-blueprint (DRAFT)
-This blueprint gives an example of how you can leverage the Genesys Open Messaging API to develop a simple bidirectional 
-chat between a customer and a call center agent within a mobile app.  This would allow you to add or extend chat functionality
-within your company's mobile app and allow your agent to communicate directly with your customer.
+# Develop chat functionality for mobile apps using Open messaging integration in Genesys Cloud and AWS services
 
-![Open Messaging Implementation](blueprint/images/ionic-mobile-open-messaging-blueprint-architecture.png "Open Messaging Implementation")
+> View the full [Develop chat functionality for mobile apps using Open messaging integration in Genesys Cloud and AWS services](https://developer.mypurecloud.com/blueprints/) article on the Genesys Cloud Developer Center.
```

Comment:
Shouldn't the link go directly to the blueprint instead of the blueprint listing page?

Reply:
once the blueprint is published, this will be updated with the exact link.